### ISSUE TITLE
"Prepare Crypto-Node.js Release" workflow fails to prepare PR

### ISSUE THREAD

**Issue by AndrewFerr** [matrix-org/matrix-rust-sdk#1793]
The `prep-crypto-nodejs-release.yml` workflow uses `peter-evans/create-pull-request@v4` to prepare a PR to merge the draft release commit back into `main`, but it hits this error that prevents the PR from being made:
```
/usr/bin/git -c protocol.version=2 fetch --no-tags --progress --no-recurse-submodules --unshallow --force origin create-pull-request/patch:refs/remotes/origin/create-pull-request/patch
fatal: couldn't find remote ref create-pull-request/patch
Pull request branch 'create-pull-request/patch' does not exist yet.
```
This happened during my [manual dispatch of the workflow](https://github.com/matrix-org/matrix-rust-sdk/actions/runs/4699305325/jobs/8332613523#step:11:65), which succeeded at cutting the release but not filing a PR (which I since [filed manually](https://github.com/matrix-org/matrix-rust-sdk/pull/1779)).

There shouldn't be anything in the draft release commit that would have impacted this, so I assume the PR-generating workflow broke in some way at some point.

**Comment by Hywan** [comment]
Yeah, this preparing script is very fragile… I really don't like it :-(.

**Comment by AndrewFerr** [comment]
Closing since the Node bindings have their own repo now.
